frappe/desk/doctype/event/event.json

```diff
@@ -272,12 +275,27 @@
    "label": "Sender",
    "options": "Email",
    "read_only": 1
+  },
+  {
+   "default": "0",
+   "depends_on": "eval:(doc.sync_with_google_calendar && in_list([\"Event\", \"Meeting\", \"Other\"], doc.event_category))",
```

Comment:
let add_video_conferencing show on all event categories? - we show the sync_with_google_calendar checkbox by default anyway (?)

```suggestion
   "depends_on": "eval:doc.sync_with_google_calendar",
```